### DISCUSSION TITLE
Map view that lays out all photos on the world map

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#563] [Enhancements]
Our desktop apps aren't at feature parity yet in terms of how we leverage location data to make information more accessible.

Lack of a Map view is one of them.

[Leaflet](https://github.com/Leaflet/Leaflet) seems like a great fit for the job, but we should first enhance the current search suggestions widget to plug in Maps as an option.

**Comment by EmPeWe** [comment]
Is there any timeline about this topic? Web and desktop are sadly lacking this feature more than 15 month. In the meantime, a hint [for the customer](https://help.ente.io/photos/features/map#how-to-view-album-photos-on-the-map) should be add, that the feature is currently not available for the mentioned platforms.

**Comment by jrmgx** [comment]
This feature is missing very badly

**Comment by boston-black** [comment]
I thought this feature would be alread here, as it can be set in the menu.

**Reply by EmPeWe** [reply]
Map view of **all** photos is not the same as location information from the image detail. The popup itself

<img width="366" height="332" alt="Bildschirmfoto 2025-09-19 um 10 38 59" src="https://github.com/user-attachments/assets/1cae3bc2-78c1-40f0-b797-216c8c2303fe" />

is misleading, because there is no "world map", but only a location info (mini map) per single image. If I touch this small image info on a mobile device, I can see real world map with all my images. On desktop/web, this is not possible, only the map could be zoomed in/out, but no other images are visible. To sum it up: yes the setting is available on desktop/web, but the feature "world map with all images" is not

**Comment by MikeScanlan5** [comment]
Yea +10000000  ... can we get the images on a map on desktop/browser? That's my main use case for these photos. :)

**Comment by basicuser01** [comment]
I need map view and facial recognition on the desktop app for Linux. The local AI sorting is basically the main use case for ente for me.

**Comment by EmPeWe** [comment]
@vishnukvmd Is there any news on this topic? Are there any plans to implement this feature on the desktop? By the way, the [help page](https://ente.io/help/photos/features/search-and-discovery/map-and-location) is still incorrect and does not distinguish between desktop/web/mobile.

**Reply by mnvr** [reply]
There are plans, yes! But no exact timelines.

About the help page, it was recently fixed - https://github.com/ente-io/ente/pull/7663/files - and now the instructions for the all photos map view is only for mobile. Maybe it needs further clarification ig.

**Comment by AswinAsok** [comment]
Just in time for this thread's 2nd anniversary, Map view is now live on web and in nightly builds for desktop! - #8238 

<img width="1707" height="1068" alt="Screenshot 2025-12-11 at 11 26 02 AM" src="https://github.com/user-attachments/assets/b0c85647-950d-4be5-bf7e-2493a7517e31" />
@EmPeWe @jrmgx @boston-black @basicuser01 @MikeScanlan5 the wait is over

**Reply by EmPeWe** [reply]
Congrats! I'm really happy to see this feature is now available on all frontends. And even the help page is up to date.
